hhs_oauth_server/settings/dev.py

```diff
@@ -8,6 +8,8 @@
 INVITE_REQUEST_ADMIN = env(
     'DJANGO_INVITE_REQUEST_ADMIN', 'change-me@example.com')
 
+ALLOW_CHOOSE_LOGIN = True
```

Comment:
I'm not exactly sure why you added this, but I wanted to let you know that this is for local development, and not related to dev.bluebutton.cms.gov. If this is intentional, that's fine, but if you want this to be true on dev, you'll have to add that in bluebutton-web-deployment.

Reply:
yup it was intentional, I wanted to leave developers working locally the option to use local auth.

Reply:
Fair enough. I don't use this settings file, although I probably should.